CONTRIBUTING.md

```diff
@@ -0,0 +1,103 @@
+# Contributing to Tombolo Digital Connector
+
+Thank you for choosing to contribute to this project. **We need your superpowers!**  
+To learn more about the Tombolo vision checkout our [website](http://www.tombolo.org.uk/).
+
+Before starting to contribute have a look at the [README](README.md) and [docs](documentation). Our suggestion is to 
+start with the [system architecture](documentation/System-Architecture.md) and [local datastore](documentation/Local-Datastore.md), and 
+eventually go further in the documentation with other topics of your interest.
+
+## Help wanted
+[Open Source Community - Digital Connector](https://github.com/FutureCitiesCatapult/TomboloDigitalConnector/milestone/22)
+milestone contains issues we think are interesting for you to work on. You can find them also through 
+[`help wanted`](https://github.com/FutureCitiesCatapult/TomboloDigitalConnector/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) 
+tag. Feel free to check the other milestones in case you find something that excites you more.
+If none of the issues reflects your problem/feature create a new one following the 
+[issue template](https://github.com/FutureCitiesCatapult/TomboloDigitalConnector/blob/master/.github/ISSUE_TEMPLATE.md).
+Include screenshots and error log if available. 
+
+## In progress
+Chosen the issue you want to work on, let us know by tagging one of the project maintainers: *@thanosbnt*, 
+*@sassalley*, *@lorenaqendro*, *@arya-hemanshu* and change the issue tag from `help wanted` to `in progress`. Regular
+ update on its status is much appreciated, so that everyone knows its proceeding and eventually help.
```

Comment:
- I don't think it is possible for `contributors` to change the issue labels.
- I think most will tag the first person or all, may be give each maintainer an area to look after?

---
e.g 

Once you have started to work on an issue, let us know by putting in a comment and tagging the relevant maintainer: 

| Maintainer | Section / Type of work |
| :--- | :--- |
| `@thanosbnt` | this section / type of work |
| `@sassalley` | this section / type of work |
| `@lorenaqendro` | this section / type of work |
| `@arya-hemanshu` | this section / type of work | 

Regular updates on the status will be useful to the team, and will allow us to not only provide support but also to understand the progress.